server/notification-providers/serwersms.js

```diff
@@ -27,6 +29,13 @@ class SerwerSMS extends NotificationProvider {
 
             let resp = await axios.post(url, data, config);
 
+            if (!resp.data.success) {
+                // If unsuccessful, try using group_id
```

Comment:
No other notification provider does work this way. Lets not make this into an odd duckling.
Retrying like this seems counterintuitive to me.

=> Please use a selection (default=phone) in the fronted which allows users to choose if they want to enter `serwersmsPhoneNumber` or `serwersmsGroupId`.
If their docs allow setting both fields at the same time, allow setting of both fields at the same time.